### PR TITLE
Update EIP-4527: Move to draft

### PR DESCRIPTION
**Pull request by aaronisme**
Update some mirror change to the EIP and move it into review

### CODE REVIEW

**Comment by aaronisme** [comment]
@axic, @SamWilsn, @Pandapip1 would you please review this change? And I am not sure whether it is ok to move stagnant to review. I checked the EIP-1, from my understanding it is ok to do it.

File: EIPS/eip-4527.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: QR Code data transmission protocol for the offline wallets
 description: QR Code data transmission protocol between wallets and offline signers.
 author: Aaron Chen (@aaronisme), Sora Lee (@soralit), ligi (@ligi), Dan Miller (@danjm), AndreasGassmann (@andreasgassmann), xardass (@xardass), Lixin Liu (@BitcoinLixin)
 discussions-to: https://ethereum-magicians.org/t/add-qr-code-scanning-between-software-wallet-cold-signer-hardware-wallet/6568
-status: Stagnant
+status: Review
```
Comment (Pandapip1) [inline EIPS/eip-4527.md:7]: Must be moved back to draft, since it never made it to review before it was stagnated.

Comment (aaronisme) [reply EIPS/eip-4527.md:7]: OK

Comment (aaronisme) [reply EIPS/eip-4527.md]: @Pandapip1 Hi, I have moved it back to draft, please check it out.

File: EIPS/eip-4527.md [hunk 2]
```diff
@@ -4,17 +4,20 @@ title: QR Code data transmission protocol for the offline wallets
 description: QR Code data transmission protocol between wallets and offline signers.
 author: Aaron Chen (@aaronisme), Sora Lee (@soralit), ligi (@ligi), Dan Miller (@danjm), AndreasGassmann (@andreasgassmann), xardass (@xardass), Lixin Liu (@BitcoinLixin)
 discussions-to: https://ethereum-magicians.org/t/add-qr-code-scanning-between-software-wallet-cold-signer-hardware-wallet/6568
-status: Stagnant
+status: Draft
```
Comment (Pandapip1) [inline EIPS/eip-4527.md:7]:
```suggestion
status: Draft
```
This is a small typo, but it blocks this PR.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
The GitHub links should be removed (if unnecessary) or their contents moved to the EIP's assets folder and updated to link there.

File: EIPS/eip-4527.md (hunk 2 above)
Comment (aaronisme) [reply EIPS/eip-4527.md]: Hi @Pandapip1 I have updated the file, would you please check it again?

File: EIPS/eip-4527.md [hunk 3]
```diff
@@ -186,35 +201,47 @@ eth-signature-bytes = bytes .size 65; the signature of the signing request (r,s,
...
 ## Reference Implementation
-This reference implementation is in Javascript and available at [https://github.com/KeystoneHQ/keystone-airgaped-base/tree/master/packages/ur-registry-eth]
+
+This reference implementation is in Javascript and available at: https://github.com/KeystoneHQ/keystone-airgaped-base/tree/master/packages/ur-registry-eth
 
 Metamask has adopted it for its integration with QR-based Signers. https://github.com/MetaMask/metamask-extension/pull/12065
 
+BlockchainCommons Uniform Resources: https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-005-ur.md
+
+BlockchainCommons UR Types: https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-006-urtypes.md
```
Comment (Pandapip1) [inline EIPS/eip-4527.md:237]: These links must be removed

File: EIPS/eip-4527.md [hunk 4]
```diff
@@ -186,35 +201,47 @@ eth-signature-bytes = bytes .size 65; the signature of the signing request (r,s,
...
+
 Here is a video that shows how it works: https://www.youtube.com/watch?v=1eM53TYG1YA
```
Comment (Pandapip1) [inline EIPS/eip-4527.md:239]: This one too

File: EIPS/eip-4527.md [hunk 5]
```diff
@@ -186,35 +201,47 @@ eth-signature-bytes = bytes .size 65; the signature of the signing request (r,s,
...
+Currently, the UR has an active `airgapped wallet community` which continues to improve the UR forward.
 
 ## Backwards Compatibility
+
 Currently, there is no existing protocol to define data transmissions via QR Codes so there are no backward compatibility issues that needs to be addressed now.
 
 ## Test Cases
-The reference implementation contains the [test cases](https://github.com/KeystoneHQ/keystone-airgaped-base/tree/master/packages/ur-registry-eth/__tests__).
+
+The reference implementation contains the `test cases`: https://github.com/KeystoneHQ/keystone-airgaped-base/tree/master/packages/ur-registry-eth/__tests__
```
Comment (Pandapip1) [inline EIPS/eip-4527.md:227]: Must be removed

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

File: EIPS/eip-4527.md (hunk 3 above)
Comment (aaronisme) [reply EIPS/eip-4527.md]: removed

File: EIPS/eip-4527.md (hunk 4 above)
Comment (aaronisme) [reply EIPS/eip-4527.md:235]: removed

File: EIPS/eip-4527.md (hunk 5 above)
Comment (aaronisme) [reply EIPS/eip-4527.md]: removed

**Comment by aaronisme** [comment]
@Pandapip1  I have fixed the comments as requested, Please check thanks

**Comment by Skylerspencer** [comment]
Thank you.

File: EIPS/eip-4527.md [hunk 6]
```diff
@@ -186,35 +201,43 @@ eth-signature-bytes = bytes .size 65; the signature of the signing request (r,s,
...
+The test cases can be found on the `ur-registry-eth` package released by the Keystone team.
 
 ## Reference Implementation
-This reference implementation is in Javascript and available at [https://github.com/KeystoneHQ/keystone-airgaped-base/tree/master/packages/ur-registry-eth]
+
+The reference implementation can be found on the `ur-registry-eth` package released by the Keystone team.
 
 Metamask has adopted it for its integration with QR-based Signers. https://github.com/MetaMask/metamask-extension/pull/12065
 
 Here is a video that shows how it works: https://www.youtube.com/watch?v=1eM53TYG1YA
```
Comment (Pandapip1) [inline EIPS/eip-4527.md:235]:
```suggestion
```

Comment (aaronisme) [reply EIPS/eip-4527.md]: removed

**Comment by aaronisme** [comment]
@Pandapip1 I have updated the file as requested

**Review by Pandapip1 — APPROVED** [review]

**Review by poxelman — APPROVED** [review]

**Pull request merged**